.github/workflows/cleanup-artifacts.yml

```diff
@@ -0,0 +1,55 @@
+name: Cleanup Artifacts
+
+on:
+  workflow_dispatch:
+    
+  schedule:
+    - cron: '30 4 * * 5'
+            
+jobs:
+    jfrog:
+        name: "Cleanup Jfrog hpccpl-docker-local"
+        runs-on: ubuntu-22.04
+        if: ${{ github.repository == 'hpcc-systems/HPCC-Platform' }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                submodules: false
+            - name: Login to JFrog
+              uses: docker/login-action@v3
+              with:
+                registry: ${{ secrets.JFROG_REGISTRY }}
+                username: ${{ secrets.JFROG_USERNAME }}
+                password: ${{ secrets.JFROG_PASSWORD }}
+            - name: Setup JFrog CLI
+              uses: jfrog/setup-jfrog-cli@v3
+              with:
+                version: 2.56.0
+              env:
+                JF_URL: https://${{ secrets.JFROG_REGISTRY }}
+                JF_USER: ${{ secrets.JFROG_USERNAME }}
+                JF_PASSWORD: ${{ secrets.JFROG_PASSWORD }}
+
+            - name: 'Cleanup release-candidates over 1 month old'
+              shell: bash
+              run: |
+                jf rt del --quiet --spec ${{ github.workspace }}/dockerfiles/rule-hpccpl-docker-local-rcs-over-1mo.spec |& tee -a rc-deletion.log
+
+            - name: 'Cleanup point releases over 6 months old'
+              shell: bash
+              run: |
+                declare -a versions=("8.6" "8.8" "8.10" "8.12" "9.0" "9.2" "9.4" "9.6")
```

Comment:
Minor:  We could pre-emptively add 9.8 and 9.10?

Reply:
Ah good idea, I believe we can. It likely will just be a success on no-op. I'll double check and add it if possible.